toplevel/coqargs.ml

```diff
@@ -179,6 +175,10 @@ let warn_deprecated_inputstate =
   CWarnings.create ~name:"deprecated-inputstate" ~category:"deprecated"
          (fun () -> Pp.strbrk "The inputstate option is deprecated and discouraged.")
 
+let warn_deprecated_boot =
+  CWarnings.create ~name:"deprecated-boot" ~category:"noop"
+         (fun () -> Pp.strbrk "The -boot is a noop.")
```

Comment:
I think should be rephrased since -boot has still the effect of not loading the rc file (like passing `-q`).
The message should say to use -q instead.

Reply:
Indeed great catch! The first version of this patch made `-boot` a noop but @SkySkimmer suggested indeed to keep it operational, however I didn't update the warning. Rebasing.